haas/migrations/versions/57f4c30b0ad4_added_metadata.py

```diff
@@ -0,0 +1,35 @@
+"""added metadata
+
+Revision ID: 57f4c30b0ad4
+Revises: 89630e3872ec
+Create Date: 2016-11-08 08:36:01.183860
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '57f4c30b0ad4'
+down_revision = '89630e3872ec'
+branch_labels = ('haas',)
+
+
+def upgrade():
+    # commands auto generated by Alembic - please adjust! #
+    op.create_table('metadata',
+                    sa.Column('id', sa.Integer(), nullable=False),
+                    sa.Column('label', sa.String(), nullable=False),
+                    sa.Column('value', sa.String(), nullable=True),
```

Comment:
Given that we no longer have separate register/update calls I don't think this makes sense. The API doesn't provide a way to clear it without deleteing the object anyway, so if this is ever null it's a bug.